controllers/v1/solutions.js

```diff
@@ -281,7 +281,11 @@ module.exports = class Solutions extends Abstract {
     /**
     * @api {post} /project/v1/solutions/addRolesInScope/:solutionId Add roles in solutions
     * @apiVersion 1.0.0
+<<<<<<< HEAD
+    * @apiName add roles in scope
+=======
     * @apiName Add roles in scope
+>>>>>>> project-template-creation-changes
```

Comment:
@Prajwal17Tunerlabs  remove these . why we need

Reply:
removed those unwanted characters